app/views/common/_pagination.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="form-group">
+  <%=
+
+    select_tag('per_page',
+                 options_for_select(per_page_options || Setting[:entries_per_page]),
```

Comment:
I don't think the setting is needed here, it will never be used (as `per_page_options` will always return a non-false value)